### PR TITLE
example unsigned CoRIM

### LINKED ISSUE #83: add an example CoRIM
as per subject, starting with the simplest case (unsigned)

### PR DESCRIPTION
**Pull request by thomas-fossati**
Fixes #83

### CODE REVIEW

File: examples/corim-unsigned-1.diag [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/ corim / 47110(
```
Comment (yogeshbdeshpande) [inline examples/corim-unsigned-1.diag:1]: I was planning to add the required header on a similar lines, using signed-corim-map ?

/ protected-signed-corim-header-map / {
/ corim.alg-id / 1 : 5,
/ corim.content-type / 3 : "application/rim+cbor",
/corim.issuer-key-id / 4 : b'0123456789',
/ corim.meta / 8 : { 
    / corim.signer / 0 : [
            {
            / corim.entity-name / x: "Acme Inc",
            / corim.reg-id / y: "http://acme.inc",
            /corim.role / z: 1 / corm.manifest-creator /
            },
            {
            / corim.entity-name / x: "Acme Inc",
            /corim.role / z: 2 / corm.manifest-signer /
            }
            ] 
        },
    },

Comment (thomas-fossati) [reply examples/corim-unsigned-1.diag:1]: This example is in a file named `corim-unsigned-1.diag` :-) We will add a signed CoRIM in a separate `corim-signed-1.diag` example.

Comment (yogeshbdeshpande) [reply examples/corim-unsigned-1.diag:1]: Great!
I agree, I will mark the conversation as resolved!

**Review by nedmsmith — APPROVED** [review]
It would be nice to have an example endorsed value triple that included svn.

**Comment by thomas-fossati** [comment]
> It would be nice to have an example endorsed value triple that included svn.

I'll iterate on this

**Pull request merged**